### PR TITLE
new csv parser

### PR DESCRIPTION
**Pull request by touv**
try to add new loader...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,8 @@
     "copy-webpack-plugin": "4.0.1",
     "csv-parse": "1.1.10",
     "expect": "1.20.2",
+    "ezs": "^1.1.3",
```
Comment (djhi) [inline package.json]: Please fix the version: don't use `^` nor `~`

File: src/api/loaders/csv.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+  stream
+  .pipe(ezs('CSVParse'))
```
Comment (djhi) [inline src/api/loaders/csv.js]: Just a note but indenting would make this easier to read

**Review by djhi — CHANGES_REQUESTED** [review]

File: src/api/loaders/csv.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+  stream
```
Comment (djhi) [inline src/api/loaders/csv.js]: Better but you should configure eslint for your IDE. This project has a specific eslint configuration with tabs made of 4 spaces :)

**Review by djhi — CHANGES_REQUESTED** [review]
Can you prefix your PR's title with either `[WIP]` or `[RFR]` so that we know if/when you're done with it ?

File: src/api/loaders/csv.js (hunk 3 above)
Comment (touv) [reply src/api/loaders/csv.js]: ok, but where is the specific config file, I don't find any file like this https://github.com/Inist-CNRS/lodex-v1/blob/master/.eslintrc.js

Comment (touv) [reply src/api/loaders/csv.js]: found !!! 
https://github.com/Inist-CNRS/lodex/blob/master/src/api/.eslintrc

**Review by djhi — APPROVED** [review]

**Pull request merged**